hedera-node/src/main/java/com/hedera/services/state/submerkle/ExpirableTxnRecord.java

```diff
@@ -109,6 +111,8 @@ public class ExpirableTxnRecord implements FCQueueElement {
 	private List<FcTokenAssociation> newTokenAssociations = NO_NEW_TOKEN_ASSOCIATIONS;
 	private ByteString alias = MISSING_ALIAS;
 	private byte[] ethereumHash = MISSING_ETHEREUM_HASH;
+	private String pseudoRandomBitString = EMPTY_STRING;
```

Comment:
I don't understand why we use an intermediate `String` here; the "true" type is `byte[384]`, right?

Reply:
The HIP says it is a `384-bit array`, so converted to binary string representation.

Reply:
Fixed !